BayPass/View/PurchesedTicketCell.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  PurchesedTicketCell.swift
+//  BayPass
+//
+//  Created by Zhe Li on 2/19/19.
+//  Copyright © 2019 Tim Roesner. All rights reserved.
+//
+
+import SnapKit
+import UIKit
+
+class PurchesedTicketCell: UITableViewCell {
+    var ticketView = TicketView(agency: "ACE", icon: UIImage(named: "CalTrain")!, cornerRadius: 8)
```

Comment:
I think using `ticketView: TicketView` here should work, then you don't need to create a ticketView with fake properties